airbyte-lib/airbyte_lib/caches/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""Base module for all caches."""
+
+from airbyte_lib.caches.base import SQLCacheBase
+from airbyte_lib.caches.duckdb import DuckDBCache, DuckDBCacheConfig
+from airbyte_lib.caches.memory import InMemoryCache, InMemoryCacheConfig
+from airbyte_lib.caches.postgres import PostgresCache, PostgresCacheConfig
+from airbyte_lib.types import SQLTypeConverter
+
+
+# We export these classes for easy access: `airbyte_lib.caches...`
+__all__ = [
+    "DuckDBCache",
+    "DuckDBCacheConfig",
+    "InMemoryCache",
+    "InMemoryCacheConfig",
+    "PostgresCache",
+    "PostgresCacheConfig",
+    "SQLCacheBase",
+    "SQLTypeConverter",
```

Comment:
Good question. Originally I wanted it public because I'm still in the mode of "connectors" receiving contributions - and/or being BYO. But this is only needed (in theory) for overriding behaviors in different Caches, so I guess we can hide it because we don't expect users to build/customize/debug those cache implementations.

Reply:
Update: Now removed from the imports surfaced in this "caches" module.